### PR TITLE
Sprint 141 blog post

### PR DESCRIPTION
**Pull request by eduardoj**

### CODE REVIEW

**Review by hennevogel — COMMENTED** [review]
Maybe you can explain a bit how those two request types hang together?

File: _posts/development/2023-06-01-request-workflow-redesign.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Request Page Redesign - Maintenance Action Types
+category: development
+---
+
+After a while keeping working on the request page redesign, we are glad to come back to you now supporting maintenance incident requests and maintenance release requests.
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply _posts/development/2023-06-01-request-workflow-redesign.md:7]: I agree. Suggestion applied.

**Comment by eduardoj** [comment]
> Maybe you can explain a bit how those two request types hang together?

Sure. I created a small section explaining the basics of both requests types, including a link to the OBS User documentation. Please take a look again.

File: _posts/development/2023-06-01-request-workflow-redesign.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+After a while keeping working on the request page redesign, we are glad to come back to you with two new request types: maintenance incident requests and maintenance release requests.
+
+The request redesign is part of [the beta program](/2018/10/04/the-beta-program/).
+
+{% include partials/_series-of-posts-about-request-redesign.md %}
+
+
+### What are Maintenance Incident Requests and Maintenance Release Requests?
```
Comment (danidoni) [inline _posts/development/2023-06-01-request-workflow-redesign.md:14]:
```suggestion
### What are Maintenance Incident and Maintenance Release Requests?
```

Comment (eduardoj) [reply _posts/development/2023-06-01-request-workflow-redesign.md:14]: I prefer to be extra clear here and leave it like it is. Do you find this is repetitive in excess?

Comment (danidoni) [reply _posts/development/2023-06-01-request-workflow-redesign.md:14]: okay, np

**Review by danidoni — APPROVED** [review]

**Pull request merged**